packages/lms/src/routes/+layout.svelte

```diff
@@ -20,11 +20,34 @@
   const scrollIntoView = (node: HTMLElement) => {
     node.scrollIntoView();
   };
+
+  let isMobile = false; // Initialize the visibility state
+
+  // Check if the code is running in a browser environment
+  if (typeof Svelte !== 'undefined' && Svelte.window) {
+    isMobile = Svelte.window.innerWidth <= 768;
+
+    // Add a listener to resize events to toggle the navigation based on viewport width
+    const handleWindowResize = () => {
+      isMobile = Svelte.window.innerWidth <= 768;
+    };
+
+    Svelte.window.addEventListener('resize', handleWindowResize);
+  }
+
+  // Declare the handleNavItemClick function
+  const handleNavItemClick = () => {
+    if (isMobile) {
+      isNavVisible = !isNavVisible; // Toggle the visibility of the navigation bar
+    }
+  };
+
+  let isNavVisible = false; // Initialize the visibility state
 </script>
 
-<Drawer>
+<Drawer open={isNavVisible}>
```

Comment:
Import the drawerStore from skeleton using my previous suggestion
```suggestion
<Drawer>
```